aspnetcore/performance/response-compression/sample/ResponseCompressionSample.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
AFAIK this is what its supposed to look like. :smile: I've been following them somewhat closely and my professor Dr. @dasmulli has been keeping an eye on me. I'll update when they get back to us here.